dxr/static/templates/path_line.html

```diff
@@ -4,7 +4,7 @@
 {%- if not is_binary or is_dir -%}
 <a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
 {%- endif -%}
-    {{ display_path }}
+    {{ display_path|safe }}
```

Comment:
Yeah, this is added in [JS](https://github.com/mozilla/dxr/blob/1666883ed0d28a7d34280c00dd477beac365f6b0/dxr/static/js/dxr.js#L119-L126). Someone could conceivably put html tags in their folder names and we would render them as markup.